File: LorePdks.COMMON/DTO/Pdks/PdksHareketDTO.cs
using LorePdks.COMMON.DTO.Base;
using LorePdks.COMMON.DTO.Hareket;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LorePdks.COMMON.DTO.Pdks
{


    public class PdksHareketDTO : BaseDTO
    {
        public PdksDTO pdksDto { get; set; }
        public HareketDTO hareketDto { get; set; }

    }
}

File: LorePdks.WEB/src/app/core/models/PdksHareketDto.ts
import { baseDto } from "./baseDto";
import { HareketDto } from "./HareketDto";
import { PdksDto } from "./PdksDto";

/**
 * Pdks hareket bilgisini temsil eden model
 */
export class PdksHareketDto extends baseDto {
  pdksDto: PdksDto;
  hareketDto: HareketDto;
}